src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -32,6 +32,8 @@
   Each nested query increments this counter by 1."
   0)
 
+(def source-table-alias "t1")
```

Comment:
If I understand this correctly, you're changing the code to give every table in a source query an alias of `t1`. What happens if something with a `JOIN` is used as a source query? Wouldn't this change break those queries?

Reply:
Yes, this is a big change, and it needs to be tested with other drivers. However, my testing indicates that joins are fine, and even enabling `:nested-queries` for Drill and Spark SQL still passes all tests.
Even if we `JOIN` we should get (at worst) something like `select a from sometable t1 join (select b, c from othertable t1) _metabase_join_alias where (t1.a == _metabase_join_alias.b)` which is still ok.

At least, that's what I think. Again, this needs to be tested with all drivers (most of which I do not have access to).